src/riak_core_claimant.erl

```diff
@@ -719,7 +719,7 @@ all_members() ->
 -spec get_remote_type_status(BucketType::binary()) ->
     {AllProps::[any()], BadNodes::[node()]}.
 get_remote_type_status(BucketType) ->
-    rpc:multicall(all_members() -- [node()],
+    rpc:multicall(all_members(),
```

Comment:
Given that we only call this from one spot in the claimant, I'd like to see the function renamed.